dev-java/java-service-wrapper/java-service-wrapper-3.5.50.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source test"
+inherit java-pkg-2 java-ant-2 toolchain-funcs
+
+MY_PN="wrapper"
+MY_P="${MY_PN}_${PV}_src"
+DESCRIPTION="A wrapper that makes it possible to install a Java Application as daemon"
+HOMEPAGE="https://wrapper.tanukisoftware.org/"
+SRC_URI="https://${MY_PN}.tanukisoftware.org/download/${PV}/${MY_P}.tar.gz"
```

Comment:
pkgcheck complains about:
```
UnstableOnly: for arch: [ arm ], all versions are unstable: [ 3.5.25-r2, 3.5.50 ]
  PotentialStable: version 3.5.25-r2: slot(0), stabled arches: [ amd64, x86 ], potential: [ ~arm ]
  SSLCertificateError: version 3.5.50: SRC_URI: SSL cert error: HTTPSConnectionPool(host='wrapper.tanukisoftware.org', port=443): Max retries exceeded with url: /download/3.5.50/wrapper_3.5.50_src.tar.gz (Caused by SSLError(CertificateError("hostname 'wrapper.tanukisoftware.org' doesn't match either of '*.tanukisoftware.com', 'tanukisoftware.com'"))): https://wrapper.tanukisoftware.org/download/3.5.50/wrapper_3.5.50_src.tar.gz
```

Reply:
That's strange, I can't get my pkgcheck to do the same, even with `-f no`.  But looking at their website, I think they changed from .org -> .com.  Try this.